Handler/Plan.hs

```diff
@@ -0,0 +1,22 @@
+module Handler.Plan where
+
+import Import
+
+getPlansR :: Handler Html
+getPlansR = do
+    plans <- runDB $ selectList [] [Asc PlanSort]
+    Entity _ user <- requireAuth
```

Comment:
I'm confused by the `_` here. Can you call the `Entity` constructor and leave off an argument?